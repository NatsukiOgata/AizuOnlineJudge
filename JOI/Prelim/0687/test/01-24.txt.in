100
323 360 350 345 336 361 330 388 339 397 352 332 318 362 392 324 377 321 334 371 389 399 370 325 319 307 379 308 329 381 376 364 348 372 393 395 351 374 338 302 342 309 346 369 365 312 340 317 394 359 304 341 301 355 383 390 331 367 314 315 349 310 305 327 356 391 384 311 333 385 400 378 368 373 354 316 387 328 343 357 303 398 366 386 344 380 306 353 326 347 358 337 396 382 335 375 363 322 320 313
