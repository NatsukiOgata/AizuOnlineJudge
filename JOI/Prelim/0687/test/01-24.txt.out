24436
10214
